src/pages/Environments.js

```diff
@@ -426,16 +429,23 @@ const Environments = () => {
             headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
               const disk = filteredDisks[rowIndex]
-              const galaxyDiskNames = _.map(disk => disk.name, galaxyDisks)
-              const runtimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames), disks)
-              const runtimeDiskNames = _.map(disk => disk.name, runtimeDisks)
-              const multipleRuntimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames) || disk.status === 'Deleting',
-                disksByProject[disk.googleProject]).length > 1
-              const multipleGalaxyDisks = _.remove(disk => _.includes(disk.name, runtimeDiskNames) || disk.status === 'DELETING',
+              const appDiskNames = _.map(disk => disk.name, appDisks)
```

Comment:
Could it be
```suggestion
              const appDiskNames = _.get('name', appDisks)
```
?

Reply:
It'll still need to me `map`, `get` would just be for one object. That said,
```suggestion
              const appDiskNames = _.map('name', appDisks)
```

Reply:
Ugh, to me `_.map('name', appDisks)` is so much harder to understand than the more `explicit _.map(disk => disk.name, appDisks)`.